src/holo-test

```diff
@@ -39,9 +39,9 @@ run_testcase() {
     # setup chroot for holo run
     rm -rf -- target/
     cp -R source/ target/
+    mkdir -p target/run
+    mkdir -p target/tmp
     mkdir -p target/usr/share/holo/files
-    mkdir -p target/usr/share/holo/run-scripts
-    mkdir -p target/usr/share/holo/users-groups
```

Comment:
I thought about that.  Since the exit-code and empty-directory changes will require the testcases for `holo-run-scripts` and `holo-users-groups` to be updated, I figured that another change on them wouldn't be too bad (if I had left those lines in, then they would have needed to appear in the `expected-tree`). Because holo-test is meant to be someone generic, I figured that it should be the responsibility of the `holo-run-scripts` and `holo-users-groups` tests to create those directories.

Since git doesn't commit empty directories, it could be done either by creating them in `env.sh`, or by putting `.keep` file in them so that git sees them.